src/briefcase/commands/new.py

```diff
@@ -327,6 +410,43 @@ def build_app_context(self):
         # The module name can be completely derived from the app name.
         module_name = self.make_module_name(app_name)
 
+        default_source_dir = f"src/{module_name}"
+        if target and target.exists():
+            source_dir = self.input_text(
+                intro=f"""
+We are attempting to set up an existing project with Briefcase (since the
+target directory exists). To do this, we need to know the path of the
+application entry point. This path should be relative to the target ({target})
+that contains a ``__main__.py`` file.
+
+For example, if you have an existing project, myapp, and the path
+supplied to --target points to the root directory of this application (e.g. myapp)
+and if you can start myapp by running ``myapp/src/myapp/__main__.py``,
+then you should set the source directory to ``src/myapp``.""",
+                variable="source directory",
+                default=default_source_dir,
+                validator=self.create_source_dir_validator(target, app_name),
+            )
+
+            test_source_dir = self.input_text(
+                intro=f"""
+Since we are attempting to set up an existing project with Briefcase, we also
+need to know the path to the test suite. This path should be relative to the
+target ({target}) and it should exist.
+
+For example, if you have an existing project, myapp, and the path supplied to
+--target points to the root directory of this application (e.g. myapp) and the
+tests are located in ``myapp/tests``, then you should set test source directory
+``tests``.
+""",
+                variable="test source directory",
+                default="tests",
+                validator=self.create_test_source_dir_validator(target),
+            )
```

Comment:
There's an edge case here of a project that doesn't have tests. This question will require the existence of an actual tests folder.

Reply:
Our thought here was that it's better to always ask the user for a test directory that exists since this will prevent a misspelling (e.g. writing test instead of tests) from breaking the briefcase setup. If a project has no tests, can the user create an empty tests directory? What do you think? Should the wizard create an empty test directory if it doesn't exist and show a warning instead?

Reply:
Validation is useful for catching the mis-spelling of a tests folder that *exists*, but it doesn't help with the "there aren't any tests to find" case. 

In both cases, we also need to do something about the test bootstrapping script. Briefcase requires that the test script has a termination sentinel, which existing test suites won't have.

I feel like we need to either (a) have an additional "Does you have a test suite?" question, or (b) allow "" as a valid answer to the question for the "I don't have tests" case. If there's no test suite, then we can copy in the version that has been templated; if they *do* have a test suite, we can copy just the bootstrap `tests/appname.py` script.

Reply:
That makes sense! We have now updated the code so it supports the second option: "" for "I don't have tests". In that case, the whole test source directory is copied. If they have tests, only `tests/appname.py` is copied and otherwise, the full test-dir is copied.